andela/ah-orcas-frontend
# Create navbar and footer

#### What does this PR do?
```it creates a navbar and footer ```
#### Description of Task to be completed?
```This PR creates a Navbar and Footer components for consistnecy header and footer ```
#### How should this be manually tested?
- Clone this repository 
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
``` cd ah-orcas-frontend```
- Checkout the setup redux branch
``` git checkout ft-navbar-footer ```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser

#### Any background context you want to provide?
- This components will helps us to have a consistency design and also makes our app scalable.

#### What are the relevant pivotal tracker stories?
`none`
#### Screenshots
![image](https://user-images.githubusercontent.com/30722290/49384050-39ca1700-f72b-11e8-8b23-96eeaecde3cd.png)

#### Checklist:
